frontend/src/metabase/ui/components/icons/Icon/Icon.tsx

```diff
@@ -39,7 +40,17 @@ export const Icon = forwardRef<SVGSVGElement, IconProps>(function Icon(
     />
   );
 
-  return tooltip ? <Tooltip label={tooltip}>{icon}</Tooltip> : icon;
+  return tooltip ? (
+    <Tooltip
+      style={{ whiteSpace: multiline ? "unset" : "nowrap" }}
+      label={tooltip}
+      data-testid={`${multiline ? "wrapped-" : ""}tooltip`}
```

Comment:
`expect(tooltip).toHaveStyle` seems like a smart way to go - great find. Is the testid still useful here?

Reply:
So, it turns out the test failed on CI, not sure why, so I changed it and it now uses the test id.